lib/util/http/index.ts

```diff
@@ -67,7 +67,7 @@ export class Http {
       ],
     };
     const res = await got(resolvedUrl, combinedOptions);
-    return { body: res.body, headers: res.headers };
+    return { body: res?.body, headers: res?.headers };
```

Comment:
In which cases can `res` be `null` or `undefined`?

Reply:
When testing with empty mocked responses. But I think I'll populate those mocks instead.